NAMESPACE

```diff
@@ -5,11 +5,29 @@ S3method(clean_quo,quosures)
 S3method(quo_chr,list)
 S3method(quo_chr,quosure)
 export("%>%")
+export(calculate_tdm)
 export(clean_column_names)
 export(clean_quo)
 export(quo_chr)
 export(unclean_column_names)
 export(write_csv_file)
+importFrom(dplyr,all_of)
+importFrom(dplyr,arrange)
+importFrom(dplyr,case_when)
+importFrom(dplyr,contains)
+importFrom(dplyr,distinct)
+importFrom(dplyr,filter)
```

Comment:
I think @AlexAxthelm prefers that `dplyr::filter()` is always namespaced, not sure exactly why but I it must conflict with some other common `filter()` function

Reply:
`base::filter`

Reply:
```r
base::filter
# Error in get(name, envir = ns, inherits = FALSE) : 
#   object 'filter' not found
```
?

Reply:
ah, nevermind `stats::filter`

Reply:
there's a `base::Filter` too 🤯